src/find-options/FindOptionsWhere.ts

```diff
@@ -40,11 +41,20 @@ export type FindOptionsWhereProperty<
           | Property
     : Property | FindOperator<Property>
 
+/**
+ * Helper type to conditionally allow null/undefined based on TypeORMSettings
+ */
+type NullableWhereValue<T> = TypeORMSettings extends {
+    allowNullableWhere: true
+}
+    ? T | null | undefined
```

Comment:
I think this type should only allow `null`, not `undefined` as well.

`null` values potentially become valid when using `sql-null` mode, but `undefined` values are always invalid regardless of the mode.

Reply:
Great point, changed it 🙏

Reply:
I just realized I didn't push the changes properly 😦 
now I pushed them and you should see them in Github 🙏